spinalcordtoolbox/deepseg/models.py

```diff
@@ -76,6 +76,14 @@
         "description": "Grey/white matter seg on exvivo human T2w.",
         "contrasts": ["t2"],
         "default": False,
+    },
+    "model_7T_multiclass_gm_sc_unet2d": {
```

Comment:
```suggestion
    "model_7t_multiclass_gm_sc_unet2d": {
```
i suggest to keep everything in small letter for consistency